src/test/java/com/targomo/jackson/datatype/trove/deser/TIntFloatMapDeserializerTest.java

```diff
@@ -0,0 +1,51 @@
+package com.targomo.jackson.datatype.trove.deser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.joda.JodaModule;
+import com.targomo.jackson.datatype.trove.TroveModule;
+import gnu.trove.map.TIntFloatMap;
+import gnu.trove.map.TIntIntMap;
+import gnu.trove.map.hash.TIntFloatHashMap;
+import gnu.trove.map.hash.TIntIntHashMap;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+public class TIntFloatMapDeserializerTest extends BaseTroveDeserializerTest
+{
+    @Test
+    public void testSimple() throws IOException
+    {
+
+        TIntFloatMap map = reader.forType(TIntFloatMap.class)
+                .readValue("{\"8\" : 1.0, \"1\" : 2.5}");
+        assertEquals(2, map.size());
+        assertThat(map.get(8)).isEqualTo(1.0f);
+        assertThat(map.get(1)).isEqualTo(2.5f);
+        assertThat(map.get(3)).isEqualTo(Integer.MAX_VALUE);
```

Comment:
why would this return int.max_value?

Reply:
because that was set for "no_value". (see https://github.com/targomo/targomo-jackson-datatype-trove/blob/bbad1f6b03edde8f0e55a6a89fee86272c216efb/src/test/java/com/targomo/jackson/datatype/trove/deser/BaseTroveDeserializerTest.java#L16)
but perhaps it's better to make this a constant for the tests

Reply:
have it as a constant in the test now